lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1105,21 +1105,22 @@ def wait_for_restart_to_finish(self, wait_time):
         Args:
             wait_time: (int) max time to wait for services to become active
         """
-        print(
-            f"Waiting for a maximum of {wait_time} "
-            f"seconds for restart to finish",
-        )
-        if self._init_system == InitMode.DOCKER:
+        if self._init_system in (InitMode.DOCKER, InitMode.SYSTEMD):
+            print(
+                f"Waiting for a maximum of {wait_time} "
+                f"seconds for restart to finish",
+            )
             start_time = time.time()
```

Comment:
Why are docker and `systemd` cases handled the same here? `check_if_magma_servcices_are_active()` does not wait for `systemd` services become functional, but returns active as soon as they started to run. This is in contrast to the docker health check that was implemented there.

Reply:
`check_if_magma_servcices_are_active` calls `systemctl is-active `

Reply:
We probably want to have notify for sctpd, pipelined and sessiond as well in the future. Currently this probably works because mme is running After=sessiond,pipelined,sctpd,.....

Reply:
Yes, that is probably the reason. Because just checking `systemctl is-active` did not do the trick before and it would be good to explicitly set `notify` to all magma services. This out of scope, but a nice follow-up ticket.